src/main.js

```diff
@@ -1,7 +1,7 @@
 // The Vue build version to load with the `import` command
 // (runtime-only or standalone) has been set in vue.config.js with runtimeCompiler.
 import Vue from 'vue';
-import Vuetify from 'vuetify';
+import Vuetify from 'vuetify/lib/framework';
```

Comment:
In the documentation from https://v2.vuetifyjs.com/en/features/treeshaking/#vuetify-loader they import vuetify from 'vuetify/lib'. Is this something that you changed or is this the default provided by the template? I understand that this probably doesn`t make any difference to bundle sizes after treeshaking - see https://stackoverflow.com/questions/67615517/vuetify-difference-between-import-vuetify-lib-framework-vs-vuetify-lib. Is there any performance difference regarding build time?

Reply:
Hi @fschmenger, concerning this one, the template generated when adding `Vuetify` to a plain `Vue` app makes use of the `vuetify/lib/framework` indeed which is fun as the template was written by `Vuetify` team... so it seems they did things differently in their docs and examples...

So that's the simple reason why I sticked to this one when I wrote this PR. Moreover, the contents of the file `vuetify/lib/index.js`, is the following:

```javascript
export * from './components';
export * from './directives';
export { default as colors } from './util/colors';
export { default } from './framework';
//# sourceMappingURL=index.js.map
```

which means that in our case, the result is the same as we import the default. This would be different if we wanted to import components separately as explained [here](https://v2.vuetifyjs.com/en/features/treeshaking/#manually-importing).

To further report on your comment, I just did some performance tests and I must admit I don't really understand all the results...

At first, I'm unable to reproduce the facts about bundle size that is explained in the very interesting link to stackoverflow you provided. Forcing bundle in development mode, here is the result while importing `framework` directly :

![image](https://github.com/wegue-oss/wegue/assets/57899415/7c4d4e5c-a8e7-49cb-8452-a480a06f736e)

and here is the results of importing from `lib` :

![image](https://github.com/wegue-oss/wegue/assets/57899415/455bec53-4e01-4041-bad0-4d31a0017e6c)

As we are now using the V2.7.X version of Vuetify, I suppose they changed something compared to what was observed in the SO post which should have been done with V2.5.X or V2.6.X... 

Now the interesting part is that I ran the build step multiple times, in development and production modes, with the import from `lib` and from `lib/framework` and I clearly see that building while importing from `framework` can get faster in dev and in prod modes !  Where I get a mean of 42 seconds to build when importing from `lib`, it gets down to 36 seconds when importing from `framework` !  The funny point is that it goes 15% faster to give the exact same result !  I suppose the import from `lib` creates a dependency graph which is more complicated but resolves the same in the end...

Reply:
Yeah, this makes sense. Probably the 3 extra imports from `lib` will just be redundant when using defaults and eventually the treeshaker has to eliminate them again?!
Anyway, thanks for looking into this. I think since this is the template default and also gives better build performance we stick with it and come back to it later, if there is any requirement to do so.
I will merge then.